code/game/machinery/doors/firedoor.dm

```diff
@@ -296,6 +296,7 @@
 			close()
 
 /obj/machinery/door/firedoor/crush()
+	. = ..()
```

Comment:
No do not call . = ..() if you want to override the proc else it will now call door/crush first just remove the . = ..() out of this and it should be fine. Might have caused a missunderstanding before sorry about that i just dropped the . = () as an example for how the parent would even ever get called.
Without . = () at this trough it should work fine.